src/Exporter.php

```diff
@@ -684,6 +684,45 @@ protected function initAttributeValues()
         return $this;
     }
 
+    /**
+     * @return array
+     */
+    protected function getRequiredVariationValues()
+    {
+        $returnArray = [];
```

Comment:
A variable named `$returnArray` could mean anything. Consider renaming this to something that matches it's semantics.